src/modules/Elsa.MassTransit.AzureServiceBus/Features/AzureServiceBusFeature.cs

```diff
@@ -82,22 +82,27 @@ public override void Configure()
                     configurator.ConcurrentMessageLimit = options.ConcurrentMessageLimit;
                     
                     configurator.UseServiceBusMessageScheduler();
-                    configurator.SetupWorkflowDispatcherEndpoints(context);
                     ConfigureServiceBus?.Invoke(configurator);
                     var instanceNameProvider = context.GetRequiredService<IApplicationInstanceNameProvider>();
 
-                    if (!massTransitFeature.DisableConsumers)
+                    // Temporary consumers are used for pub/sub, which is used for invalidating local caches and should therefore not be disabled.
+                    foreach (var consumer in temporaryConsumers)
```

Comment:
I considered the cancellation event, and figured it might not be problematic if that runs on a non-worker role, but maybe it can be a sufficient amount of work in case of mass & cascading cancellations. So I agree, I'll add a property or attribute thing.

Reply:
> We would probably need to apply the same logic to the loop on line 68

Even when adding the consumers, they will not get activated if they're not bound to an endpoint. At the same time, you may be right that it's therefore unnecessary to register them in the first place. I'll have a look.

Reply:
Done. We can now control precisely what consumers should remain in tact, regardless of the DisableConsumers feature setting.